docs/source/tutorials/datalab/workflows.ipynb

```diff
@@ -929,7 +944,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This section focused on identifying null values, but `Datalab` can detect a variety of other issues such as (near) duplicates, outliers, label issues and more. Explore our additional tutorials to learn more about how Datalab can enhance your data quality workflows.\n",
+    "This section focused on identifying null values, but `Datalab` can detect a variety of other issues such as (near) duplicates, outliers, label issues and more. Explore our additional tutorials to learn more about how `Datalab` can enhance your data quality workflows.\n",
     "\n",
```

Comment:
```suggestion
```

Reply:
unclear what value that section is providing, seems a bit obvious at the point in a user journey where they are reading this part and finding this null section useful in the first place.

Reply:
That is a valid point! This can be removed.